rage/atArray.hpp

```diff
@@ -20,7 +18,16 @@ namespace rage
             m_size(0),
             m_count(0)
         {
-            
+
+        }
+
+        atArray(const atArray& right)
+        {
+            m_count = right.m_count;
+            m_size = right.m_size;
+
+            m_data = (T*)tlsContext::get()->m_allocator->Allocate(m_size * sizeof(T), 16, 0);
+            std::uninitialized_copy(right.m_data, right.m_data + right.m_count, m_data);
         }
```

Comment:
Guard it with `#ifdef _WIN32`, the Linux build has `tlsContext::get()` ifdef'd out.

Same with the other places using `tlsContext::get()`

Reply:
What is it supposed to use then? This function will be called when the atArray needs to expand.

Reply:
Just ifdef everything out that wouldn't work on Linux, no one will be making a Linux native internal mod menu for GTA V anyways. If they plan on compiling a Windows binary under linux these functions will become available (aka. it satisfies the _WIN32 define). These classes are mostly made available for linux externals, in which you wouldn't be able to use these functions either way (since `tlsContext::get()` is something strictly internal only).

Reply:
Word.